### PR TITLE
Describe `--conf` and a few minor fixes

### PR DESCRIPTION
**Pull request by rshym**
Add missing info regarding `--conf` command-line option.

Fix few misprints and replace tabs to spaces.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,13 @@ Depending on your Linux configuration, you might need to run
 
 #### Choose your config
 
+By default `rcar_flash` reads all required data from shipped `rcar_flash.yaml`
```
Comment (lorc) [inline README.md:49]: I believe this part should go into beginning of  "Reference manual" section. We don't need this in quickstart.

File: README.md [hunk 2]
```diff
@@ -262,7 +271,7 @@ HyperFlash or eMMC. Those "programs" are simple lists of "wait-for" -
 
 This example describes how to flash QSPI on s4/spider
 board. `rcar_flash` will wait for ">", then send "xls2", then wait for
-"(1-3)" message, chose option "1" and so on...
+"(1-3)" message, choose option "1" and so on...
```
Comment (lorc) [inline README.md:274]: Thank you for fixes, but would you please move them into a separate commit?

**Review by lorc — COMMENTED** [review]
With comments addressed:

`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by rshym** [comment]
- Fix of misprints is moved to a separate commit.
- Description of `--conf` is moved to "Reference manual"
- `Ack` is added.

Ready to be merged.
